packages/core/src/node/synchronizer.ts

```diff
@@ -679,12 +676,12 @@ export class Synchronizer extends EventProcessor {
       limit: 1,
     })
     const fromBlock = lastMassDeposit[0]?.blockNumber || proposedAt
-    const pivotBlock =
-      (await this.l1Contract.provider.getBlockNumber()) -
-      this.blockCache.BLOCK_CONFIRMATIONS
+    const pivotBlock = await this.l1Contract.provider.getBlockNumber()
```

Comment:
remove `BLOCK_CONFIRMATIONS`, the reason as comment below, to avoid a gap from `L1 block number` to `L1 block number + BLOCK_CONFIRMATIONS`

Reply:
Yes, this is one of the reasons to make lower performance of the coordinator.
but this delayed checking makes a more stable network between L1 and coordinators.

I think this still need on zkopru testnet goerli. and also we can reduce the block confirmation variable by coordinator configuration.